app/controllers/user_sessions_controller.rb

```diff
@@ -22,6 +22,8 @@ def create
       if @user_session.save
         flash[:notice] = ts("Successfully logged in.")
         @current_user = @user_session.record
+        REDIS_GENERAL.sadd("last_login_list", params[:user_session][:login])
+        REDIS_GENERAL.set("last_login_#{params[:user_session][:login]}", Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.